chapter_appendix_math/geometry-linear-algebric-ops_vn.md

```diff
@@ -1082,15 +1082,16 @@ M_inv.dot(M)
 ### Numerical Issues
 -->
 
-### *dịch tiêu đề phía trên*
+### Vấn đề số học
```

Comment:
Numerical thường được dùng để nói về các vấn đề liên quan đến tính toán của máy tính, chẳng hạn như làm tròn hay tràn số.
```suggestion
### Vấn đề tính toán
```